charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.71.2
```

Comment:
only bump the patch version please, we'll bump the minor version and app version in a separate PR.

Reply:
What will happen if a feature gate that doesn't exist in a specific version is enabled? Will the manager fail?

Reply:
the chart was just bumped so the gate should exist now